packages/aws-cdk/lib/api/cloudformation-deployments.ts

```diff
@@ -152,6 +153,14 @@ export interface DeployStackOptions {
    * @default - nothing extra is appended to the User-Agent header
    */
   readonly extraUserAgent?: string;
+
+  /**
+   * Allows adding CloudWatch log groups to the log monitor via
+   * hotswapLogMonitor.addLogGroups();
+   *
+   * @default - not monitoring CloudWatch logs
+   */
+  readonly hotswapLogMonitor?: CloudWatchLogEventMonitor;
```

Comment:
We need to remove the word "hotswap" from these property names, as this monitor is now not related to hotswapping anymore.

Reply:
renamed to `cloudWatchLogMonitor`